addon/components/power-calendar/days.js

```diff
@@ -78,7 +78,7 @@ export default Component.extend({
         daysOfWeek = daysOfWeek.filter((d) => d.isCurrentMonth);
       }
       weeks.push({
-        id: days[0].moment.format('YYYY-w'),
+        id: days[i].moment.format('YYYY-w'),
```

Comment:
Wouldn't that be identical? I mean, all days of the same week belongs to the same week, so it doesn't matter which day you use, right?